package/MDAnalysis/analysis/hbonds/wbridge_analysis.py

```diff
@@ -551,46 +622,57 @@ def run(self, **kwargs):
                 self._update_water_selection()
 
             s1_frame_results_dict = {}
-            if self.selection1_type in ('donor', 'both') and self._water_acceptors:
+            if self.selection1_type in ('donor', 'both') and \
+            self._water_acceptors:
                 self.logger_debug("Selection 1 Donors <-> Water Acceptors")
                 ns_acceptors = AtomNeighborSearch(self._water_acceptors)
                 for i, donor_h_set in self._s1_donors_h.items():
                     d = self._s1_donors[i]
                     for h in donor_h_set:
                         res = ns_acceptors.search(h, self.distance)
                         for a in res:
-                            donor_atom = h if self.distance_type != 'heavy' else d
+                            donor_atom = h if self.distance_type != 'heavy' \
```

Comment:
keep on same line

I know, I asked you to use 79 characters, but readability also counts. It's important to know when to slightly break the rules.